### PR TITLE
Add EditorConfig file

### PR DESCRIPTION
**Pull request by AlexWayfer**
More info [here](http://editorconfig.org/).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{md,markdown}]
+trim_trailing_whitespace = false
+
+[test/mocks/*.txt]
+indent_size = 3
```
Comment (panthomakos) [inline .editorconfig]: This seems unnecessary. Don't particularly care how these files are formatted.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (panthomakos) [inline .editorconfig]: The convention in this project is to not insert a final newline. Otherwise the settings look good to me.

Comment (AlexWayfer) [reply .editorconfig]: No! There are final newlines throughout the project!

`lib/timezone.rb` at your `master`:

image

[And it's good practice for UNIX](https://stackoverflow.com/a/729795/2630849) (try to `cat lib/timezone.rb` with newline and without).

GitHub will display files without final newlines abnormally, with the crossed out symbol like this `↲` ("Enter") at the last line.

File: .editorconfig (hunk 1 above)
Comment (AlexWayfer) [reply .editorconfig]: OK, done.

File: .editorconfig (hunk 2 above)
Comment (panthomakos) [reply .editorconfig]: Is this setting adding an additional blank line or just a newline on line 79?

Comment (AlexWayfer) [reply .editorconfig]: > Is this setting adding an additional blank line or just a newline on line 79?

These are equivalent things :grimacing: 

Adding newline at 79 makes blank (empty) 80 line.

Try to `$ cat lib/timezone.rb` with newline (and 80 line, which doesn't show in GitHub, for example) and without these things.

**Pull request merged**